tests/integration_tests/prompts/test_jinja2.py

```diff
@@ -0,0 +1,171 @@
+"""Test functionality related to prompts."""
+from typing import Dict, List, Tuple
+
+import pytest
+
+from langchain.prompts.few_shot import FewShotPromptTemplate
+from langchain.prompts.prompt import PromptTemplate
+
+
+def test_prompt_from_jinja2_template() -> None:
```

Comment:
These were using optional dependencies (jinja2). 

I'm not sure if we want to have something that runs more frequently than integration tests to test this?

Reply:
Was this picked up from poetry as well?

Reply:
oh nevermind -- misunderstood your comment